engine-precompiles/src/modexp.rs

```diff
@@ -22,6 +22,9 @@ impl<HF: HardFork> ModExp<HF> {
     fn calc_iter_count(exp_len: u64, base_len: u64, bytes: &[u8]) -> Result<U256, ExitError> {
         let start = usize::try_from(base_len).map_err(utils::err_usize_conv)?;
         let exp_len = usize::try_from(exp_len).map_err(utils::err_usize_conv)?;
+        if exp_len > usize::MAX - 96 {
```

Comment:
It seems it would be good to introduce a new constant `const EXP_LEN: usize = usize::MAX - 96;`

Reply:
And leave a comment about what a magic 96 is.

Reply:
This PR will be merged into #689 in one PR.